conf/squid/common/ms_3way_5node_1client_rh.yaml

```diff
@@ -0,0 +1,147 @@
+globals:
+  - ceph-cluster:
+      name: ceph-pri
+      node1:
+        role:
+          - _admin
+          - installer
+          - mon
+          - mgr
+          - crash
+      node2:
+        role:
+          - mon
+          - mgr
+          - crash
+          - rgw
+          - osd
+        no-of-volumes: 6
+        disk-size: 40
+      node3:
+        role:
+          - mon
+          - mgr
+          - crash
+          - rgw
+          - osd
+        no-of-volumes: 6
+        disk-size: 40
+      node4:
+        role:
+          - mon
+          - mgr
+          - crash
+          - rgw
+          - osd
+        no-of-volumes: 6
+        disk-size: 40
+      node5:
+        role:
+          - mon
+          - mgr
+          - crash
+          - rgw
+          - osd
```

Comment:
Also please add rbd-mirror role to all three clusters in this config as well.

Reply:
added